tests/link/jax/test_sparse.py

```diff
@@ -2,13 +2,51 @@
 import pytest
 import scipy.sparse
 
+
+jax = pytest.importorskip("jax")
+from jax.experimental.sparse import BCOO
+
 import pytensor.sparse as ps
 import pytensor.tensor as pt
 from pytensor import function
-from pytensor.graph import FunctionGraph
+from pytensor.graph import Constant, FunctionGraph
+from pytensor.tensor.type import DenseTensorType
 from tests.link.jax.test_basic import compare_jax_and_py
 
 
+def assert_bcoo_arrays_allclose(a1, a2):
+    assert isinstance(a1, BCOO)
+    assert isinstance(a1, BCOO)
+    np.testing.assert_allclose(a1.todense(), a2.todense())
+
+
+@pytest.mark.parametrize("sparse_type", ("csc", "csr"))
+def test_sparse_io(sparse_type):
```

Comment:
This seems like it is testing the sparse transpose `Op` as well, the name should reflect that or the transpose test should be split into a separate test

Reply:
The sparse transpose Op is just a tool, not the thing of interest in this test (we need to have some sparse operation, otherwise it would be testing a not-implemented sparse Copy Op).